src/test/java/seedu/address/logic/parser/AddItemCommandParserTest.java

```diff
@@ -84,18 +109,18 @@ public void parse_compulsoryFieldMissing_failure() {
                 expectedMessage);
 
         // missing quantity prefix
-        assertParseFailure(parser, ITEM_NAME_DESC_BANANA
+        /*assertParseFailure(parser, ITEM_NAME_DESC_BANANA
                         + VALID_ITEM_QUANTITY_BANANA
                         + ITEM_DESCRIPTION_DESC_BANANA
                         + ITEM_LOCATION_DESC_PEACH_ORCHARD,
-                expectedMessage);
+                expectedMessage);*/
 
         // missing description prefix
-        assertParseFailure(parser, ITEM_NAME_DESC_BANANA
+        /*assertParseFailure(parser, ITEM_NAME_DESC_BANANA
                         + ITEM_QUANTITY_DESC_BANANA
                         + VALID_ITEM_DESCRIPTION_BANANA
                         + ITEM_LOCATION_DESC_SPINACH_GARDEN,
-                expectedMessage);
+                expectedMessage);*/
```

Comment:
change these tests to using wrong prefix
can remove the old tests once replaced